hwdb.d/80-ieee1394-unit-function.hwdb

```diff
@@ -850,6 +850,13 @@ ieee1394:ven000050C2mo00000070sp000050C2ver00000001
  ID_MODEL_FROM_DATABASE=Duende FireWire
  IEEE1394_UNIT_FUNCTION_AUDIO=1
 
+ieee1394:node:ven0x00022emo0x800006units0x00022e:0x800006
+ieee1394:ven0000022Emo00800006sp0000022Ever00800006
+  ID_VENDOR_FROM_DATABASE=TASCAM
+  ID_MODEL_FROM_DATABASE=IF-FW/DM mkII
+  IEEE1394_UNIT_FUNCTION_MIDI=1
+  IEEE1394_UNIT_FUNCTION_AUDIO=1
```

Comment:
Please use single space indentation, there seem to two.

Reply:
> Please use single space indentation, there seem to two.

Oops. I forgot absolutely that the format of hwdb uses one space indentation... Furthermore, some bugs of my tool brings invalid output for the modalias. I correct them and force-pushed the renewed change.

Thanks for your reviewing.